source/includes/_advanced-usage.md

```diff
@@ -44,3 +44,40 @@ Your React components hierarchy should look as following:
   <DialogOpener />
 </TipserElementsProvider>
 ```
+
+## Embedding Elements in native apps
+
+Tipser Elements is a web-based library and currently there is no native version for Android or iOS available.
+
+However it is possible to embed Elements in a native app with some amount of custom code. We recommend using one of the patterns described below.
+
+### Pattern 1: Full embedded integration
+
+For each of e-commerce subpages/articles in your app do the following:
+
+1. Create a dedicated WebView screen for that page and link it to your application (make sure the user has a way to go back to the native part of application) 
+2. Build and host a web page based on Elements, following the instructions from [Tipser Elements](#tipser-elements) or [Tipser Elements React](#tipser-react-elements)
+3. Point your WebView to the corresponding web page
+
+*Disabling dialogs with redirects*
+
+The default behavior of opening product dialogs when a product tile is clicked may not be desired in your mobile app. If that's the case, you can replace that behavior with
+redirecting to a product page by setting the configuration option: `disableDialog` to `true`. One of the benefits is that the back button will work in the way expected by users
+(getting moving back from product page to the e-commerce page).
+
+### Pattern 2: Checkout-only integration
+
+If you prefer even more native experience, you may use [Tipser Rest API](https://developers.tipser.com/rest-api) to fetch the product and shopping cart and build
+your own native implementation of product view, product tiles and cart icon components. Your implementation will need to use [Tipser SDK](#tipser-sdk) to add products to cart:
```

Comment:
how would they use the Tipser sdk in their native code ? Or did you mean that they have to host some webpage anyway and run Tipser sdk there? Or you mean, you that they have to semi native codebase like react-native to use our tipser-sdk ?

Reply:
Could you provide some details here @piotr-sobczyk about how the native code would run Tipser SDK ?

Reply:
@chmurson you are right, this may not be a precise descripition. After discussing it with @mateuszsikora we think there are 3 possible solutions:
1. Everytime they add to cart, they do it via SDK in a "hidden" iframe (I'm not 100% sure if that's possible)
2. They keep the state of the cart (list of the product ids) in the native code and only share it with us just before opening the checkout dialog (they must do it in a WebView anyway)
3. We provide them native version of SDK (like the one we started writing during the hackathon)